File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/contagionrecordop/i/IContagionRecordQryService.java
package iih.ci.rcm.contagionrecordop.i;

import iih.ci.rcm.contagionrecordop.d.ContagionRecordOpDTO;
import xap.mw.core.data.BizException;
import xap.sys.appfw.orm.handle.dataobject.paging.PaginationInfo;
import xap.sys.appfw.orm.handle.dataobject.paging.PagingRtnData;
import xap.sys.appfw.tmpl.qryscheme.qrydto.QryRootNodeDTO;

public interface IContagionRecordQryService {
	public abstract PagingRtnData<ContagionRecordOpDTO> getOpContagionQryDTOList(
			QryRootNodeDTO qryRootNodeDTO, PaginationInfo paginationInfo)
			throws BizException;

	public abstract ContagionRecordOpDTO[] getAllPageData() throws BizException;
}

File: client/iih_stub/iih.ci_stub/iih.ci.rcm/contagionrecordop/i/IContagionRecordQryService.cs
using iih.ci.rcm.contagionrecordop.d;
using xap.mw.core.data;
using xap.sys.appfw.tmpl.qryscheme.qrydto;

namespace iih.ci.rcm.contagionrecordop.i
{
    public interface IContagionRecordQryService {
        PagingRtnData<ContagionRecordOpDTO> getOpContagionQryDTOList(QryRootNodeDTO qryRootNodeDTO, PaginationInfo paginationInfo);

        ContagionRecordOpDTO[] GetAllPageData();
    }
}
